### PR TITLE
Mayhem Integration

### PR DESCRIPTION
**Pull request by Gediminas19**

### CODE REVIEW

File: mayhem/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+         -C llvm-args=-sanitizer-coverage-inline-8bit-counters \
+         -C llvm-args=-sanitizer-coverage-trace-geps \
+         -C llvm-args=-sanitizer-coverage-prune-blocks=0 \
+         -C debug-assertions=on \
+         -C debuginfo=0 \
+         -C opt-level=3 \
+         -Z sanitizer=address" \
+    ASAN_OPTIONS=" detect_odr_violation=0" \
+    cargo build --target x86_64-unknown-linux-gnu --bins
+
```
Comment (xansec) [inline mayhem/Dockerfile:32]: Please copy the resulting binaries into a package stage to reduce image size.

Comment (xansec) [reply mayhem/Dockerfile:32]: I see that you are also using corpora. You might want to copy the corpus directories into the package stage as well. Note that the arguments in the `cmd` section of the Mayhemfiles specifying corpora will have no effect. You'll need to specify corpus directories explicitly in the Mayhemfile with:
```
tests:
  - file:///absolute/path/to/my/corpus
```
See here for more information: https://mayhem.forallsecure.com/docs/reference/mayhemfile/#tests

**Review by xansec — CHANGES_REQUESTED** [review]
Does `rust-toolchain` need to be updated? If so, why?

**Comment by xansec** [comment]
Please update this PR or we will have to close

**Comment by Gediminas19** [comment]
Sorry, I'll do it tonight

File: mayhem/Dockerfile (hunk 1 above)
Comment (Gediminas19) [reply mayhem/Dockerfile:32]: The corpus directories are actually empty (the original fuzzing in the repo had this too), so I omitted this. Let me know if I should have added them to the Mayhemfile.

**Comment by xansec** [comment]
LGTM!

**Pull request merged**